ompy/filehandling.py

```diff
@@ -253,7 +253,7 @@ def save_numpy_2D(matrix: np.ndarray, Eg: np.ndarray,
 
 
 def load_numpy_2D(path: Union[str, Path]
-                  ) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+                  ) -> Tuple[np.ndarray, np.ndarray, np.ndarray, Union]:
```

Comment:
Plain type Union is not allowd

Reply:
This was not suppose to be there! Have removed now!

Reply:
Why should the type here be changed anyhow? It still returns three numpy arrays, right? Otherwise the return has to be changed, too.